BuildAndTest.proj

```diff
@@ -70,6 +70,7 @@
       <TestAssemblies Include="Binaries\Debug\Roslyn.Compilers.VisualBasic.Syntax.UnitTests.dll" />
       <TestAssemblies Include="Binaries\Debug\Roslyn.Compilers.VisualBasic.CommandLine.UnitTests.dll" />
       <TestAssemblies Include="Binaries\Debug\Roslyn.Diagnostics.Analyzers.FxCop.UnitTests.dll" />
+      <TestAssemblies Include="Binaries\Debug\System.Runtime.Analyzers.UnitTests.dll" />
```

Comment:
You need to delete this line.  This particular DLL is not built from Roslyn2013.sln and hence it can't be included in a CI run yet.